setup.py

```diff
@@ -9,14 +9,9 @@
 with open(path.join(here, "README.md"), encoding="utf-8") as f:
     long_description = f.read()
 
-req_path = path.join(here, "requirements.txt")
-with open(req_path, "r") as f:
-    install_reqs = f.read().strip()
-    install_reqs = install_reqs.split("\n")
-
 setup(
     name="shimmingtoolbox",
-    python_requires=">=3.7",
+    python_requires=">=3.6",
```

Comment:
Have you folks internally discussed which python version you want to start developing this package for? In AxonDeepSeg we've recently just transitioned from Python 3.6 to 3.7, and if you recommend users to use conda virtual environments then it's really not an issue to move ahead in version number.

Reply:
During the little formation, we tried to avoid conda and made them install 3.8, but I don't think we discussed it. It should be discussed since it will impact (a little bit) on portability. However, I know that 3.6 is used a lot in the Python community

Reply:
I am against developing for conda on the principle that it's a very specialized environment. Developing for stock cpython will make code that's portable to conda, but I don't think the reverse would be true, and I don't like building deep dependency trees. It's also not that hard to cover 3.6..3.9 in one codebase, there's no major syntax changes since 3.6, and we have [Travis](https://docs.travis-ci.com/user/build-matrix/) and/or [tox](https://tox.readthedocs.io/) to handle validating multiple versions.

But to be clear, I only bumped it backwards for my own convenience, because of the machine I happened to be working on had python3=python3.6.  We can scratch this change.

Reply:
I also think that working with Python 3.6 isn't a problem for the same reasons. I would keep it that way.

Reply:
Ok re:conda, it was just a suggestion! We've had better experience in AxonDeepSeg managing user experiences by requiring conda venvs, so I was just sharing our experience 🙂

Reply:
But as for making the code compatible for the 3.6 to 3.9 range - you should modify the Travis config file to test on all those versions if that's yoyr goal, otherwise some bugs might slip for users running Python 3.6 if some devs are using features only available for higher python versions.